handlers/get_tasks_next.go

```diff
@@ -0,0 +1,38 @@
+package handlers
+
+import (
+	"database/sql"
+	"log"
+
+	"github.com/gin-gonic/gin"
+)
+
+func GetTasksNext(c *gin.Context) {
+
+	db, err := sql.Open("postgres", "postgres://postgres:mysecretpassword@localhost:5432/postgres?sslmode=disable")
+
+	limit := c.Query("limit")
+	if limit == "" {
+		limit = "10"
+	}
+
+	rows, err := db.Query("SELECT * FROM tasks WHERE completed=false AND due_date<=NOW() ORDER BY due_date ASC LIMIT $1", limit)
```

Comment:
Seguro que queremos ese filtro de `due_date<=NOW()` ?  Porque si queremos las siguientes tareas, no solo queremos las pasadas, sino que queremos incluso las futuras no?

Reply:
vale, yo creo que podemos corregirlo así con un cambio, usando between en vez de <,

sugiero estos cambios para poder seleccionar los días que queremos ver tareas, pongo como predefinido en caso de error, o no indicar, 7 días:

daysAheadStr := c.Query("daysAhead")
	daysAhead, err := strconv.Atoi(daysAheadStr)
	if err != nil {
		daysAhead = 7 // usar un valor predeterminado si daysAhead no se puede convertir a entero
	}

	now := time.Now()                      // fecha actual
	future := now.AddDate(0, 0, daysAhead) // fecha futura (por ejemplo, dentro de 7 días)

	rows, err := db.Query("SELECT * FROM tasks WHERE completed=false AND due_date BETWEEN $1 AND $2 ORDER BY due_date ASC LIMIT $3", now, future, limit)